src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
```

Comment:
This depends on your text editor how to visualize tab '\t'. In Atom/Vim on my Linux box, neither of them are aligned. Both are ugly.

Personally, I'm not into clutter in the log like the double quote for message and print out how to interprete log message. Most of the time less is more.